frontends/web/src/routes/account/summary/summarybalance.tsx

```diff
@@ -107,7 +95,7 @@ export const SummaryBalance = ({
           <tbody>
             { accounts.length > 0 ? (
               coins.map(coinCode => {
-                const balanceRows = accountsPerCoin[coinCode].map(account =>
+                const balanceRows = accountsPerCoin[coinCode]?.map(account =>
```

Comment:
nit: I see that `coins` is obtained from `accountsPerCoin` keys. Do we need the `?` anyway? I think that if we have the coinCode, we should also have at least one account. wdyt? It can't hurt, anyway.